common/src/main/java/com/wynntils/core/consumers/overlays/TextOverlay.java

```diff
@@ -32,6 +33,9 @@ public abstract class TextOverlay extends DynamicOverlay {
     @Persisted(i18nKey = "overlay.wynntils.textOverlay.fontScale")
     public final Config<Float> fontScale = new Config<>(1.0f);
 
+    @Persisted(i18nKey = "overlay.wynntils.textOverlay.enabledTemplate")
+    public final Config<String> enabledTemplate = new Config<>("string_equals(world_state;\"WORLD\")");
```

Comment:
My suggestion is that the default is an empty template, and that the code use the current check if it finds an empty template.